app/config/services.yml

```diff
@@ -288,6 +292,15 @@ services:
           - '@router'
           - '@security.http_utils'
 
+    elife.journal.security.voter.feature.digest_channel:
+        class: eLife\Journal\Security\Voter\FixedVoter
```

Comment:
Means that it can be enabled/disabled per environment.

Reply:
Yes, given no backend exists yet it's important for this to be deployable as soon as an empty API can be created, and enabled on `continuumtest` for example. Will uncouple ingestion, infrastructure work and implementation of the service.